tool.go

```diff
@@ -24,6 +24,9 @@ type Context struct {
 // The path "std" is expanded to all packages in the Go standard library.
 // The path "cmd" is expanded to all Go standard commands.
 // The string "..." is treated as a wildcard within a path.
+// When matching using wilcards, directories with the prefix of dot or
```

Comment:
One correction. This doesn't affect only wildcard (typo in that word), but also others. E.g., it affects "all".

Maybe just drop the "using wildcards" part?